src/opm/parser/eclipse/EclipseState/Schedule/TimeMap.cpp

```diff
@@ -115,7 +116,8 @@ namespace {
             }
         }
 
-        for( const auto& keyword : deck ) {
+        SCHEDULESection schedule(deck);
+        for( const auto& keyword : schedule ) {
```

Comment:
I suppose you could write this even more succinctly as
```C++
for (const auto& keyword : SCHEDULESection(deck)) {
    // ...
}
```

Reply:
Done